usegalaxy.org/apollo.yml

```diff
@@ -0,0 +1,10 @@
+tool_panel_section_label: Apollo
+tools:
+- name: apollo_list_organism
```

Comment:
y'all don't need these, unless you want to deploy Apollo (you don't). sorry I should've filtered these since it only applies to AU and EU 

won't hurt anything tho they'll just be non functional which is fine

Reply:
I'll drop it, but is there a reason I don't want to run it?